tests/Feature/AnswerApiTest.php

```diff
@@ -0,0 +1,69 @@
+<?php
+
+namespace Tests\Feature;
+
+use App\Http\Requests\CreateAnswerRequest;
+use App\Models\Answer;
+use Illuminate\Foundation\Testing\DatabaseMigrations;
+use Illuminate\Testing\Fluent\AssertableJson;
+use Tests\TestCase;
+
+class AnswerApiTest extends TestCase
+{
+    use DatabaseMigrations;
+
+    public function test_new_answer_created_success(): void
+    {
+
+        CreateAnswerRequest::create(1);
```

Comment:
You don't need to do this bit, laravel will deal with making a CreateAnswerRequest based on the data you're sending with $this->postJson